pipes/WDL/tasks/tasks_assembly.wdl

```diff
@@ -204,7 +204,10 @@ task refine {
         assembly.py --version | tee VERSION
 
         ln -s ${assembly_fasta} assembly.fasta
-        read_utils.py novoindex assembly.fasta --loglevel=DEBUG
+        read_utils.py novoindex \
+        assembly.fasta \
+        ${"--NOVOALIGN_LICENSE_PATH" + novocraft_license} \
```

Comment:
these all need an extra space or `=` between `--NOVOALIGN_LICENSE_PATH` and the license file (see [CI failure](https://platform.dnanexus.com/projects/F8PQ6380xf5bK0Qk0YPjB17P/monitor/analysis/Fp724k00xf5kYpgJ57VZpq8x)).

Reply:
Thanks for the review